app/tests/conftest.py

```diff
@@ -1098,11 +1098,78 @@ def record_files():
         ),
     }
 
+    # file with format thats not supported by IIIF
```

Comment:
I feel like the tests in general (not just the ones in this PR) might be simplified by using more specific fixtures or direct setup in the tests themselves so it is easier to show that the setup is correct. I think we see the issue with having this abstract bulk record_files fixture show itself in the tests where we need to add comments to the code to explain that nth file has particular properties.

Not necessarily for this PR, but just so we keep it in mind going forward

Reply:
definitely, I feel like this issue could be solved by not returning the files as an array from the fixture, maybe key value pairs where the key is descriptive about what the file is or what its purpose it - for this ticket I'll make some fixtures that have descriptive names but I'll make a ticket to look into this further